Fix yr weather direction

MagMar94 opened the pull request:
Fixes so the Yr weather direction is not inverted. This error was [reported in the community](https://forum.magicmirror.builders/topic/17561/wind-direction-180-degrees-twisted-at-yr).

I misunderstood when developing the module because of the wind direction arrow was pointing opposite to the arrow displayed on Yr.no. I renamed the variable to help other developers in the future.

rejas commented:
Will have a look at this later when home (and if there are tests for this) Have you checked if every provider handles wind direction the same?

MagMar94 commented:
> Have you checked if every provider handles wind direction the same?

No, but I only changed the variable assignment in the Yr-provider.

I renamed the variable for everyone to make It easier to understand what value the weather module expects. If a provider returns the wrong value it will result in wrong cardinal directions.

I missed this when I implemented the provider, because I matched the arrow (`showWindDirectionAsArrow`) to what was displayed on Yr.no with the mirror. On Yr (and other weather services) arrows points in the direction the wind is blowing, but the current MM-implementation points to where the wind is blowing from. I will fix this in another PR referencing https://github.com/MichMich/MagicMirror/issues/3019. I created the issue to get some feedback before I fix it.
